ccmlib/node.py

```diff
@@ -332,7 +332,11 @@ def grep_log_for_errors(self, filename='system.log', distinct_errors=False, sear
         if search_str:
             search_str = search_str if case_sensitive else search_str.lower()
 
-        with open(os.path.join(self.get_path(), 'logs', filename)) as f:
+        log_file = os.path.join(self.get_path(), 'logs', filename)
+        if not os.path.exists(log_file):
+            return []
```

Comment:
if there is no log file it is a problem, no?

Reply:
As I explain above it may happens when we remove one cluster and create another one and parameter KEEP_DIR=False